DepartmentDetails/src/com/fsd/group4/main/Main.java

```diff
@@ -0,0 +1,58 @@
+package com.fsd.group4.main;
+
+import com.fsd.group4.departments.AdminDepartment;
+import com.fsd.group4.departments.HrDepartment;
+import com.fsd.group4.departments.SuperDepartment;
+import com.fsd.group4.departments.TechDepartment;
+
+public class Main {
+  static void displayDepartment(SuperDepartment dep) {
+    System.out.println("Welcome to " + dep.departmentName());
+  }
+
+  public static void main(String[] args) {
+
+    /*
+     * Expected Output
+     * 
+     * Welcome to Admin Department
+     * Complete your documents submission
+     * Complete by EOD
+     * Today is not a Holiday
+     * Welcome to HR Department
+     * team Lunch
+     * Fill today’s timesheet and mark your attendance
+     * Complete by EOD
+     * Today is not a Holiday
+     * Welcome to Tech Department
+     * Complete coding of Module 1
+     * Complete by EOD
+     * Core Java
+     * Today is not a Holiday
+     */
+
+    SuperDepartment adminDepartmentObject = new AdminDepartment();
```

Comment:
I did this deliberately, because superdepartment is contain all function that admin department have, so we can do it. So I applied it that I can remember it.

Reply:
can we take the instances into super department and run call through variable , check the instance to be Hr or tech to call the additional functions

Reply:
the loop can be like 

this .diaplayDepartmet(SuperDepartmentObject)
SuperDepartmentObject.getTodaysWork()
SuperDepartmentObject.getWorkDeadline()
SuperDepartmentObject.isTodayAHoliday()

then we can check for instance
SuperDepartmentObject.isInstanceOf..then call the additional methods